java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -1020,6 +1031,85 @@ public static Table readParquet(ParquetOptions opts, HostMemoryBuffer buffer,
         null, buffer.getAddress() + offset, len, opts.timeUnit().typeId.getNativeId()));
   }
 
+
+
+  /**
+   * Read an Avro file using the default AvroOptions.
+   * @param path the local file to read.
+   * @return the file parsed as a table on the GPU.
+   */
+  public static Table readAvro(File path) {
+    return readAvro(AvroOptions.DEFAULT, path);
+  }
+
+  /**
+   * Read an Avro file.
+   * @param opts various Avro parsing options.
+   * @param path the local file to read.
+   * @return the file parsed as a table on the GPU.
+   */
+  public static Table readAvro(AvroOptions opts, File path) {
+    return new Table(readAvro(opts.getIncludeColumnNames(),
+            path.getAbsolutePath(), 0, 0));
```

Comment:
Nit: Continuation indent should be 4 spaces

Reply:
Done